website/docs/tutorials/structured_config/4_defaults.md

```diff
@@ -68,7 +68,37 @@ db:
   ...
 ```
 
-#### Requiring users to specify a default list value
+### A Note about composition order
+The default composition order in Hydra is that values defined in a config are merged into values introduced from configs in the Defaults List - or in other words - overriding them.
+This behavior can be unintuitive when your primary config is a Structured Config, like in the example above.
```

Comment:
This is definitely quite unintuitive, though seeing the `_self_` makes it a bit clearer.

Reply:
I agree. Unfortunately this behavior makes a ton of sense in all other use cases and making the behavior special in this unique cases will likely add even more confusion.